README.md

```diff
@@ -96,6 +96,28 @@ One way to setup this repo is to add the following to the `~/notebooks/.user_set
 You can replace `~/notebooks/lsst-sitcom/notebooks_vandv` with the directory where this file is located.
 
 
+## Developer Guide
+
+We will try to adopt most of the practices/workflow adopted by the Telecope and Site and Data-Management teams.  
+Here are the links for both of them:  
+  
+- https://tssw-developer.lsst.io/  
+- https://developer.lsst.io/  
```

Comment:
Maybe it would be better to have hyperlinks instead of the link itself...

Reply:
I thought about it. But since the URL is so short, I decided to go keep it as it is. Let me know if this is a blocker or not.